TECH-468: Update BB Digital Registry - POST /data/<registryname>/<versionnumber>/read

KarolinaKopacz opened the pull request:
Update BB Digital Registry - POST /data/registryname/versionnumber/read

Ticket: https://govstack-global.atlassian.net/browse/TECH-468

ingmarvali commented:
Answers: 
1. one or more is ok to send.
2. Response contains all data about this record were the ID matches. 
3. Response if no answer found: 
<html>
<body>
<!--StartFragment-->

Error: response status is 404Response bodyDownload{   "detail": "no record found" }
--


<!--EndFragment-->
</body>
</html>

KarolinaKopacz commented:
@ingmarvali can I add error response to the API spec? 
Just status 404 and response body: 
`{ "detail": "no record found" }`

KarolinaKopacz commented:
@ingmarvali please review the file with the API spec because I added new 404 response. In the .json file I also fixed the formatting this is why it looks like everything in the file changed, but my changes are only in the lines 691-706.
